starter-code/src/components/Button.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import "./Button.scss"
+
+const Button = () => {
```

Comment:
In order to reduce even more amount of code, try to get used to this:
export const default Button = () => .... etc

Basically most components are getting exported, so exporting them in the same line as declaring them is a very good practise